securityscanutils/securityscan.go

```diff
@@ -157,14 +162,28 @@ func (s *SecurityScanner) GenerateSecurityScans(ctx context.Context) error {
 				return eris.Wrapf(err, "error fetching all issues from github.com/%s/%s", repo.Owner, repo.Repo)
 			}
 		}
+
+		if repo.Opts.CreateGithubIssuePerLTSVersion && len(repoLTSFilter.MaxPatchForMinor) == 0 {
```

Comment:
This seems like our implementation code is overly aware of our interface. Namely, we read properties off a specific type and then use the interface method. If we change our implementation of the repoLTSFilter, we would have to change this code as well. Ideally, it would be great if we could change the implementation of an interface instance, and not have any other dependent code change.

Reply:
I agree this isnt the cleanest part but I was to lazy to add in an additional set of post filters at this point in time and wanted to see opinions on the predicate overhaul that you had proposed earlier.